.drone.yml

```diff
@@ -96,6 +112,14 @@ build:
     when:
       success: true
 
+notify:
+  slack:
+    channel: vic-bots
+    webhook_url: $$SLACK_URL
+    username: drone
+    when:
+      repo: vmware/vic
+
```

Comment:
maybe add `branch: master`, but not many people branch off vmware/vic so not too big of a deal